### ISSUE TITLE
Auto-completion in string literals works

### ISSUE THREAD

**Issue by gavr123456789** [benwaffle/vala-language-server#106]
**Describe the bug**

**Software**
OS and version (e.g. Ubuntu 20.04): arch btw
VSCode
Vala Language Server (e.g. git commit, or PPA/AUR version): workflow-improvements branch
Vala version (valac --version): 0.48.4

**To Reproduce**
![изображение](https://user-images.githubusercontent.com/30507409/80455537-9fc4c900-8934-11ea-9048-8f8413c21593.png)

**Comment by gavr123456789** [comment]
![изображение](https://user-images.githubusercontent.com/30507409/80462113-ee776080-893e-11ea-95bc-e76f98f68521.png)
wait, actually it can be good xdd.

**Comment by Prince781** [comment]
Yeah, I can't decide if this should be a bug or not. It has its pros and cons. What do you think` @benwaffle ?

**Comment by benwaffle** [comment]
It's a bug. Strings aren't expected to have correct symbols. If you need it, vscode handles autocompletion of words found in your file.

**Comment by Prince781** [comment]
Makes sense. It seems this would be expensive to avoid from the SymbolExtractor, since this would require searching (in the worst case) the entire document for quotation marks.

**Comment by gavr123456789** [comment]
But I think it still should work in @ strings  
![image](https://user-images.githubusercontent.com/30507409/81630739-0820a980-940f-11ea-8f9b-07c227e6a2e2.png)

**Comment by Prince781** [comment]
One way to solve this is to create regions of a text document that aren't to have completions in them. These regions would be updated every time the document is compiled.
